src/libs/API.js

```diff
@@ -956,6 +956,17 @@ function Policy_Create(parameters) {
     return Network.post(commandName, parameters);
 }
 
+/**
+ * @param {Object} parameters
+ * @param {Array} [parameters.emailList]
+ * @param {String} [parameters.policyID]
```

Comment:
I don't think `policyID` or `emailList` should be optional (that's what the `[]` mean in this JSDoc).

Also we should use `requireParameters()` [like we do for this command in Web-Expensify](https://github.com/Expensify/Web-Expensify/blob/c0a9aba689e24c179bb403f1fcc59e452b926198/site/lib/api.js#L743)

Reply:
Ah, thanks for mentioning that, I thought it means something else in this case. Updated it!